raiden/blockchain_events_handler.py

```diff
@@ -59,10 +63,14 @@ def handle_tokennetwork_new(raiden, event, current_block_number):
     for channel_proxy in netting_channel_proxies:
         raiden.blockchain_events.add_netting_channel_listener(channel_proxy)
 
-    token_network_state = get_token_network_state_from_proxies(
-        raiden,
-        manager_proxy,
-        netting_channel_proxies,
+    network_graph = TokenNetworkGraphState(networkx.Graph())
+    token_address = data_decoder(event.event_data['args']['token_address'])
+
+    token_network_state = TokenNetworkState(
+        manager_address,
+        token_address,
+        network_graph,
+        [],  # TODO: Do we need this argument to even exist with the events approach?
```

Comment:
not really, we could just set it to the empty list in the constructor, same thing for the empty graph